### PR TITLE
[bitnami/node-exporter] Upgrade values.yaml for hostPID

### PR DESCRIPTION
**Pull request by micholini**
Signed-off-by: Michal Grych <m.grych@piwik.pro>

Description of the change

Set parametr hostPID to false, because of security danger. Polaris scan detected, when hostPID is set true, it's danger, because of expose the services to the host. Potentially leaking information such as environment and variables and configuration [https://kubesec.io/basics/spec-hostpid/].
Also I added officialy Polaris page [https://polaris.docs.fairwinds.com/checks/security/], where it's visibility hostPID and pottentialy dangerous.

Set hostPID it's more comfortable to have variable in values.yaml instead of set a value for each separately.

Benefits

No security danger on node-exporter.

Possible drawback

Applicable issues

Additional information

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/node-exporter/README.md [hunk 1]
```diff
@@ -130,6 +130,7 @@ The command removes all the Kubernetes components associated with the chart and
 | `updateStrategy.type`                         | The update strategy type to apply to the DaemonSet                                                                                                                 | `RollingUpdate`         |
 | `updateStrategy.rollingUpdate.maxUnavailable` | Maximum number of pods that may be made unavailable                                                                                                                | `1`                     |
 | `hostNetwork`                                 | Expose the service to the host network                                                                                                                             | `true`                  |
+| `hostPID`                                     | Allows visibility of processes on the host, potentially leaking information such as environment variables and configuration                                        | `false`                 |
```
Comment (Mauraza) [inline bitnami/node-exporter/README.md:133]: As it was previously set to true, can you set the default value to true?

File: bitnami/node-exporter/values.yaml [hunk 2]
```diff
@@ -267,6 +267,8 @@ updateStrategy:
     maxUnavailable: 1
 ## @param hostNetwork Expose the service to the host network
 hostNetwork: true
+## @param hostPID Allows visibility of processes on the host, potentially leaking information such as environment variables and configuration
+hostPID: false
```
Comment (Mauraza) [inline bitnami/node-exporter/values.yaml:271]:
```suggestion
hostPID: true 
```

**Review by Mauraza — COMMENTED** [review]
Could you check the comments? And would you check the list that appears in the PR?

File: bitnami/node-exporter/README.md (hunk 1 above)
Comment (micholini) [reply bitnami/node-exporter/README.md]: Hello, hostPID was changed to default - value is true. @Mauraza

**Pull request merged**